public/api-merged.json

```diff
@@ -14823,6 +14962,10 @@
           "type": "boolean",
           "x-go-name": "Global"
         },
+        "includeHidden": {
```

Comment:
same as above: this change in the model was introduced [here](https://github.com/grafana/grafana-enterprise/pull/2989/files#diff-4a6a200bd703d6cdc5258455569746f486c44c861ced31ea423abc18b41cb3ba)